### ISSUE TITLE
Select a "range"

### ISSUE THREAD

**Issue by aessig** [tyleryasaka/semantic-ui-range#3]
Hi, 
Thank you very much for your add-on. 
I'm looking into extending it to have a double selection that can allow to literally select a range. Do you have any suggestion ? 
Cheers.

**Comment by tyleryasaka** [comment]
@aessig Yes, I think I could definitely tell you what you would need to do or I'd even be glad to help you myself. I'm really busy right now so I don't know how soon I'll be able to take a look at it, but I can possibly get back with you in a few days.

**Comment by tyleryasaka** [comment]
@aessig I haven't caught a break yet. I will probably have a chance to work on it in a month or so.

**Comment by AlexanderYZhang** [comment]
@tyleryasaka I was thinking about taking a look at this issue to implement it for a project I am working on. Do you have any suggestions or any updates on the progress? 

Thanks!

**Comment by tyleryasaka** [comment]
@AlexanderYZhang I haven't got to it yet, but you are welcome to implement it if you like.

**Comment by xanview** [comment]
+1

Selecting a single value is nice, but is there any way to select a "range" with this "range slider"? :)

**Comment by AlexanderYZhang** [comment]
There is a range slider module being added to the official semantic-ui library here:
https://github.com/Semantic-Org/Semantic-UI/pull/4212

But for now, you can check out my fork: 
https://github.com/AlexanderYZhang/semantic-ui-range
I hacked something together that handles a range selection.

**Comment by xanview** [comment]
@AlexanderYZhang Thank you for that, I'll check it out. Do you have a demo of it in action?

**Comment by AlexanderYZhang** [comment]
Unfortunately, I'm not hosting it anywhere right now but I just added a test html file under a new branch on my repo if you want to check it out. The usage is basically the same, you just pass in an array rather than a single value.
